waiter/src/waiter/state.clj

```diff
@@ -21,6 +21,7 @@
             [metrics.meters :as meters]
             [metrics.timers :as timers]
             [plumbing.core :as pc]
+            [swiss.arrows :refer :all]
```

Comment:
Please avoid using `:refer :all`, instead use `:as sa` and qualify uses of the methods.

Reply:
We use `:refer :all` for swiss arrows in Cook.

Reply:
Maybe we shouldn't there either ;)